test/_setup.test.ts

```diff
@@ -0,0 +1,18 @@
+import { test } from 'tap';
+import {
+  ensureVirtualenv,
+  chdirWorkspaces,
+  activateVirtualenv,
+  pipInstall,
+} from './test-utils';
+
+test('install requirements in "pip-app" venv (may take a while)', async (t) => {
```

Comment:
what is this testing?

Reply:
This is setup for the tests (this is why it is in the file called `_setup.test.ts`).
This is not my idea. I have just extracted it from inspect.test.js to make sure it is run before all the tests.